tests/mocks.py

```diff
@@ -399,9 +303,7 @@ def step(
 ContinuousMAEnvironment = get_ma_environment(ContinuousEnvironment)
 
 
-class MockedMADiscreteEnvironment(
-    DiscreteMAEnvironment, DiscreteEnvironment  # type: ignore
-):
+class MockedMADiscreteEnvironment(DiscreteMAEnvironment, DiscreteEnvironment):  # type: ignore
```

Comment:
Related to the comment about removing "MA" from the class name.

Reply:
See above comment.